arbitrum-docs/how-arbitrum-works/01-a-gentle-introduction.mdx

```diff
@@ -0,0 +1,71 @@
+- Minimum cost: designed and engineered to minimize the L1 gas footprint of the system, minimizing per-transaction cost.
+
+Some other Layer 2 systems provide some of these features, but to our knowledge no other system offers the same combination of features at the same cost.
+
+Nitro is a major upgrade to Arbitrum, improving over "classic" Arbitrum in several ways:
+
+- **Advanced Calldata Compression,** which further drives down transaction costs on Arbitrum by reducing the amount of data posted to L1.
+- **Separate Contexts For Common Execution and Fault Proving,** increasing the performance of L1 nodes, and thus offering lower fees.
+- **Ethereum L1 Gas Compatibility,** bringing pricing and accounting for EVM operations perfectly in line with Ethereum.
+- **Additional L1 Interoperability,** including tighter synchronization with L1 Block numbers, and full support for all Ethereum L1 precompiles.
+- **Safe Retryables,** eliminating the failure mode where a retryable ticket fails to get created.
+- **Geth Tracing,** for even broader debugging support.
+- And many, many more changes.
+
+## The Big Picture
+
+At the most basic level, an Arbitrum chain works like this:
+
+<ImageWithCaption
+  caption="Original napkin sketch drawn by Arbitrum co-founder Ed Felten"
+  src="https://lh4.googleusercontent.com/qwf_aYyB1AfX9s-_PQysOmPNtWB164_qA6isj3NhkDnmcro6J75f6MC2_AjlN60lpSkSw6DtZwNfrt13F3E_G8jdvjeWHX8EophDA2oUM0mEpPVeTlMbsjUCMmztEM0WvDpyWZ6R"
+/>
+
+Users and contracts put messages into the inbox. The chain reads the messages one at a time, and processes each one. This updates the state of the chain and produces some outputs.
+
+If you want an Arbitrum chain to process a transaction for you, you need to put that transaction into the chain’s inbox. Then the chain will see your transaction, execute it, and produce some outputs: a transaction receipt, and any withdrawals that your transaction initiated.
+
+Execution is deterministic -- which means that the chain’s behavior is uniquely determined by the contents of its inbox. Because of this, the result of your transaction is knowable as soon as your transaction has been put in the inbox. Any Arbitrum node will be able to tell you the result. (And you can run an Arbitrum node yourself if you want.)
+
+All of the technical detail in this document is connected to this diagram. To get from this diagram to a full description of Arbitrum, we’ll need to answer questions like these:
+
+- Who keeps track of the inbox, chain state, and outputs?
+- How does Arbitrum make sure that the chain state and outputs are correct?
+- How can Ethereum users and contracts interact with Arbitrum?
+- How does Arbitrum support Ethereum-compatible contracts and transactions?
+- How are ETH and tokens transferred into and out of Arbitrum chains, and how are they managed while on the chain?
+- How can I run my own Arbitrum node or validator?
+
+## Nitro's Design: The Four Big Ideas
```

Comment:
Not sure if we want to do this in this PR, but this document (and the whole "How Arbitrum works" section) has always needed a revision based on the current [Style conventions](https://docs.arbitrum.io/for-devs/contribute#style-conventions). For example, titles should be in "sentence-case", in this case "Nitro's design: the four big ideas".

Reply:
I agree, there are Style conventions that need addressed. As this effort is mainly to restructure, to stage for the major update coming close behind (where we will enforce Style conv)